apidef/oas/operation.go

```diff
@@ -420,22 +440,28 @@ func (x *XTykAPIGateway) getOperation(operationID string) *Operation {
 	return operations[operationID]
 }
 
+// ValidateRequest holds configuration required for validating requests.
 type ValidateRequest struct {
-	Enabled           bool `bson:"enabled" json:"enabled"`
-	ErrorResponseCode int  `bson:"errorResponseCode,omitempty" json:"errorResponseCode,omitempty"`
+	// If set to `true`, it enables request validation.
```

Comment:
We should start with field name.

Reply:
According to godoc, field names aren't enforced for godoc comments:

> “Doc comments” are comments that appear immediately before **top-level package, const, func, type, and var declarations** with no intervening newlines. Every exported (capitalized) name should have a doc comment.

That doesn't really matter, because I agree with you and we should document the fields according to godoc. I've addressed it by improving the xtyk_doc generator, to verify and error out if the comment for a field doesn't start with `FieldName ` (trailing space to avoid `FieldName:`), thus catching all the violations like this.